src/notation/view/widgets/editstyle.ui

```diff
@@ -13347,25 +13356,6 @@ By default, they will be placed such as that their right end are at the same lev
   <tabstop>buttonTogglePagelist</tabstop>
   <tabstop>resetStylesButton</tabstop>
  </tabstops>
- <resources>
-  <include location="../../notationscene.qrc"/>
- </resources>
- <connections>
-  <connection>
-   <sender>pageList</sender>
-   <signal>currentRowChanged(int)</signal>
-   <receiver>pageStack</receiver>
-   <slot>setCurrentIndex(int)</slot>
-   <hints>
-    <hint type="sourcelabel">
-     <x>42</x>
-     <y>22</y>
-    </hint>
-    <hint type="destinationlabel">
-     <x>395</x>
-     <y>16</y>
-    </hint>
-   </hints>
-  </connection>
- </connections>
```

Comment:
Not sure whether this change is OK.

Reply:
hmmm... there is not supposed to be any changes here ... maybe i pressed something ... i'll double check the .ui

Reply:
I usually manually exclude changes to Ui files I didn't intend to change

Reply:
ah i see, the connection between the QListWidget and the QStackWidget is now replaced with the EditStyle::pageListRowChanged in editstyle.cpp, so this part is removed by Qt Creator

Reply:
I see, thos this change is needed.

Reply:
at leats the `connection` part, nit should about the `resources` part though

Reply:
the resources part looks like "something that should not be touched" LOL
I put it back and the feature still works.

Reply:
unnecessary changes cleaned